app/views/abuse_reports/new.html.erb

```diff
@@ -7,10 +7,59 @@
 <!--/subnav-->
 
 <!--main content-->
+<div class="userstuff">
+  <p><%= ts('The Abuse committee looks after all of your concerns regarding content and user behavior which breach the
+           Archive %{tos_link}. For technical help, please contact %{support_link}.',
+            tos_link: (link_to ts("Terms of Service"), tos_path), support_link: (link_to ts("Support"),
+                                                                                         new_feedback_report_path)).html_safe %></p>
+  <p><%= ts('<strong>We investigate every report we receive.</strong> For this reason, we ask you to not submit several
+           reports regarding any one issue unless you have additional information to offer. We also ask that you do not
+           encourage other users to submit a report regarding content you have already reported. Doing so slows down our
+           response and reaction time considerably, as we are a small volunteer-based committee.').html_safe %></p>
+  <p>
+    <%= ts('Please submit a report via this form if you have not submitted this report within the past fourteen (14) days,
+          and you:') %>
+  </p>
+  <ul>
+    <li><%= ts('would like to contact us regarding your <a href="http://www.archiveofourown.org/tos_faq#next_of_kin">Fannish Next of Kin</a>, or').html_safe %></li>
```

Comment:
I think we want to use `tos_faq_path(anchor: 'next_of_kin')` here to avoid any http/https issues down the line.